pkg/authorizer/claims.go

```diff
@@ -14,62 +15,73 @@ import (
 	"time"
 )
 
+const LabelUserClaim = "user_claim"
+const LabelOrganizationClaim = "org_claim"
+const LabelTeamClaims = "team_claims"
+const LabelDatasetClaim = "dataset_claim"
+const LabelServiceClaim = "service_claim"
+
 // Claims is an object containing claims and user info
 type Claims struct {
-	OrgClaim     organization.Claim
-	DatasetClaim dataset.Claim
-	UserClaim    user.Claim
-	TeamClaims   []teamUser.Claim
+	OrgClaim     *organization.Claim
+	DatasetClaim *dataset.Claim
+	UserClaim    *user.Claim
+	TeamClaims   []*teamUser.Claim
```

Comment:
I think this one should stay as `[]teamUser.Claim`. I think an empty or nil slice is an idiomatic  as indicating absence of team claims.

Unless there is a case I'm not thinking of. Like a case for a `TeamsClaims` value that contains both nil and non-nil claims?

Reply:
Thanks, John. I will make your suggested change. I don't think are cases for anything outside of "none" -> an empty slice (no Team Claims) or "some" -> a slice containing 1 or more Team Claims.